presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -84,6 +84,109 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Type mapping
+------------
+
+PrestoDB and PostgreSQL each support types that the other does not. When reading from or writing to PostgreSQL, Presto converts
+the data types from PostgreSQL to equivalent Presto data types, and from Presto to equivalent PostgreSQL data types.
+Refer to the following sections for type mapping in each direction.
+
+PostgreSQL type to PrestoDB type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps PostgreSQL types to the corresponding PrestoDB types following
+this table:
+
+.. list-table:: PostgreSQL type to PrestoDB type mapping
+  :widths: 50, 50
+  :header-rows: 1
+
+  * - PostgreSQL type
+    - PrestoDB type
+  * - ``BIT``
+    - ``BOOLEAN``
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+  * - ``SMALLINT``
+    - ``SMALLINT``
+  * - ``INTEGER``
+    - ``INTEGER``
+  * - ``BIGINT``
+    - ``BIGINT``
+  * - ``REAL``
+    - ``REAL``
+  * - ``DOUBLE PRECISION``
+    - ``DOUBLE``
+  * - ``NUMERIC(p, s)``
+    - ``DECIMAL(p, s)``
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+  * - ``ENUM``
+    - ``VARCHAR``
+  * - ``BYTEA``
+    - ``VARBINARY``
+  * - ``DATE``
+    - ``DATE``
+  * - ``TIME``
+    - ``TIME``
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+  * - ``TIMESTAMPTZ``
+    - ``TIMESTAMP``
+  * - ``MONEY``
+    - ``DOUBLE``
+  * - ``UUID``
+    - ``UUID``
+  * - ``JSON``
+    - ``JSON``
+  * - ``JSONB``
+    - ``JSON``
+
+No other types are supported.
+
+PrestoDB type to PostgreSQL type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps PrestoDB types to the corresponding PostgreSQL types following
+this table:
+
+.. list-table:: PrestoDB type to PostgreSQL type mapping
```

Comment:
```suggestion
.. list-table:: PrestoDB to PostgreSQL type mapping
```
Change suggested to parallel https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst